src/test/java/community/whatever/onembackendjava/application/UrlShortenServiceTest.java

```diff
@@ -0,0 +1,36 @@
+package community.whatever.onembackendjava.application;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class UrlShortenServiceTest {
+
+    @Autowired
+    private UrlShortenService urlShortenService;
+
+    @Test
+    @DisplayName("shorten-url을 생성하고 조회한다.")
+    void shortenUrlCreateAndSearch() {
```

Comment:
```suggestion
    void shorten_url을_생성하고_조회한다() {
```

`nit` 저는 이런 형태를 더 선호 합니다. 
- DisplayName 을 따로 써야되는 것은 둘째치고, 그것에 해당하는 메서드를 영작해야되는데 꽤 귀찮습니다.
- 꼭 이렇게 써야된다면 저는 메서드 이름을 그냥 test1 로 했던 것 같습니다.
- 사실 이제는 kotlin 으로 넘어와서 큰 신경 안쓰고 있습니다.